src/Qwiq.Core/Project.cs

```diff
@@ -48,11 +47,18 @@ public bool Equals(IProject other)
 
         public IWorkItemTypeCollection WorkItemTypes => _wits.Value;
 
+        public IQueryFolderCollection QueryHierarchy => _queryHierarchy.Value;
```

Comment:
This name seems odd. Maybe `SavedQueries`?

Reply:
QueryHierarchy is what vs calls it in the soap implementation, I thought about changing it to SavedQueries, but wasn't 100% interms of changing semi established naming. Still think we should change?

Reply:
Change to SavedQueries